bedrock/products/tests/test_views.py

```diff
@@ -90,7 +90,6 @@ def test_platforms(self):
             newsletters='guardian-vpn-waitlist'
         )
 
-
```

Comment:
Flake8 is complaining because there should be 2 blank lines here between classed.